nui/src/main/java/org/terasology/nui/widgets/types/math/RectanglefWidgetFactory.java

```diff
@@ -0,0 +1,98 @@
+// Copyright 2021 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+package org.terasology.nui.widgets.types.math;
+
+import org.terasology.joml.geom.Rectanglef;
+import org.terasology.nui.databinding.Binding;
+import org.terasology.nui.util.RectUtility;
```

Comment:
**suggestion:** I don't know what this is, but I'd like to get rid of it, as I think we should not need it anymore... `Rectanglef` is in our control now over in joml-ext.